src/main/java/seedu/address/logic/commands/client/DeleteClientCommand.java

```diff
@@ -26,10 +26,33 @@
 
     public static final String MESSAGE_DELETE_CLIENT_SUCCESS = "Deleted Client: %1$s";
 
+    public static final String MESSAGE_FORCE_DELETE_CLIENT_USAGE = COMMAND_WORD
+            + ": Cannot delete the Client identified by the index number because there are schedules tied to it.\n"
+            + "To force delete, pass in f/ true as an option. BEWARE, YOU WILL LOSE ALL RELATED SCHEDULES.\n"
+            + "Parameters: INDEX (must be a positive integer) f/ true\n"
+            + "Example: " + COMMAND_WORD + " 1 f/ true";
```

Comment:
It doesn't actually check for true right? Both this and delete sessions command will execute the command with just the force prefix

Reply:
yes, it works even without the true. not sure why @benclmnt also have the true in the cmd usage message

Reply:
agreed. based on our discussion in telegram, the docs should reflect this.